app/constants/assets-config.js

```diff
@@ -49,6 +49,11 @@ module.exports = {
       extensions: ['do', 'ado', 'mata'],
       categories: ['code'],
     },
+    {
+      name: 'Java',
+      extensions: ['java', 'class', 'jar'],
```

Comment:
I like having java, class, and jar files detected for Java.  What about .ear and .war?